rasa/core/utils.py

```diff
@@ -22,7 +22,7 @@
 from rasa.core.lock_store import LockStore, RedisLockStore, InMemoryLockStore
 from rasa.utils.endpoints import EndpointConfig, read_endpoint_config
 from sanic import Sanic
-from sanic.views import CompositionView
+from sanic.views import HTTPMethodView
```

Comment:
Thanks for the contribution! Can you please also adapt the usage in Line 129?

Reply:
FWIW, `HTTPMethodView` and `CompositionView` are not interchangeable.

Reply:
You probably want to change this:

```python
        if not isinstance(route.handler, CompositionView):
            handlers = [
                (list(route.methods)[0], route.name.replace("rasa.server.", ""))
            ]
        else:
            handlers = [
                (method, find_route(v.__name__, endpoint) or v.__name__)
                for method, v in route.handler.handlers.items()
            ]
```

to this:

```python
        handlers = [
            (list(route.methods)[0], route.name.replace("rasa.server.", ""))
        ]
```

Reply:
Done. Thanks.